index.js

```diff
@@ -51,6 +51,12 @@ Plugin.prototype.apply = function(compiler) {
       }
 
       var chunks = {};
+      var outputDir = this.options.path || '.';
+      var outputFilename = path.join(outputDir, this.options.filename || DEFAULT_OUTPUT_FILENAME);
+      var obj = JSON.parse(fs.readFileSync(outputFilename, 'utf8'));
+      if (obj.status == 'done') {
+          chunks = obj.chunks;
+      }
```

Comment:
Also, my fix in this PR is working fine even if I don't use an extra plugin to clear webpack-stats.json before compilation, because first it will write {"status": "compile"} to webpack-stats.json during "compile" phase, then it will call code in "done" phase for several times for each output. So it works fine.